rllib/tuned_examples/appo/pong-appo.yaml

```diff
@@ -12,10 +12,13 @@ pong-appo:
     config:
         # Works for both torch and tf.
         framework: torch
+        # Make analogous to old v4 + NoFrameskip.
         env_config:
             frameskip: 1  # no frameskip
-        vtrace: True
-        use_kl_loss: False
+            full_action_space: false
+            repeat_action_probability: 0.0  # deterministic
```

Comment:
hmm what is this. What is the repeat action probability by default and how much does it matter?

Reply:
Repeat action prob by default is 025, meaning that in 25% of all `Env.step(action)` calls, the env doesn't actually apply the given action, but repeats the previous one. This means that by default, Atari envs are stochastic, but our benchmarks always used the deterministic setting.
See here for more details: https://gymnasium.farama.org/environments/atari/#version-history-and-naming-schemes

We used to run against: "PongNoFrameskip-v4", where the v4 indicates repeat_action_probability=0.0 and the NoFrameskip means frameskip=0.